Fix "A Thief in Norg?!"  content group IDs

claywar opened the pull request:
<!-- Remove space and place 'x' mark between square [] brackets or click the checkbox after saving to affirm the following points: -->
<!-- (it should look like this: - [x] I have ...) -->
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?
* Corrects all mob spawn positions for A Thief in Norg BCNM based on retail captures
* Fixes the required group to not include Onki's elemental
* Corrects default actions for SAM quests leading up to this BCNM
* Corrects missing variable for entry requirement
* Adds jobspecial mixin for required enemies
* Adds missing entries for Ayakashi in SQL so that it will spawn correctly on Astral Flow

<!-- Describe what your PR does here. If it closes an existing issue, you can mention: "Closes #1234" and GitHub will link this PR to that issue. -->

## Steps to test these changes
Run through this battlefield in all areas.  Note: Additional issue raised to support the second avatar that uses Astral Flow in this BCNM.

<!-- Clear and detailed steps to test your changes here -->

claywar commented:
Good to go now, Ayakashi will spawn appropriately, though until a separate PR goes in, if GM is enabled, you won't get the ability triggered.

WinterSolstice8 commented on scripts/quests/outlands/Yomi_Okuri.lua:
```diff
@@ -207,16 +207,22 @@ quest.sections =
-                        (vanadielHour > 18 or vanadielHour < 5) and
-                        not GetMobByID(valkurmID.mob.DOMAN):isSpawned() and
-                        not GetMobByID(valkurmID.mob.ONRYO):isSpawned()
+                        (vanadielHour > 18 or vanadielHour < 5)
```
Is this supposed to be 19 to 4 or 18 through 5? (>= and <= or > and <)

claywar replied:
Definitely should be >=/<, good catch

claywar committed:
Fix "A Thief in Norg?!"  content group IDs

Turn off pet spawn, since mob_pets spawn automatically

Add missing variable to getVar

Fix default actions for SAM AF2, AF3

Correct spawn points, add additional NPCs for Waughroon Shrine

Add jobspecial mixin for enemies

Add missing mob entries, cleanup groups

Remove avatars from required group

Fix conditional for Yomi Okuri timing

Fix conditional again
